source/core/GenericFunctions.cpp

```diff
@@ -1782,12 +1783,32 @@ VIREO_FUNCTION_SIGNATURE1(IsLTAccumulator, void)
                 binop->_p0 = temp;
                 if (*dest) {
                     *dest = false;  // flip the result and return
-                    return null;
+                    return true;
                 }
             }
-            binop = (BinaryCompareInstruction*) next;
+            binop = (BinaryCompareInstruction*)next;
         }
     }
+    return false;
+}
+//------------------------------------------------------------
+// Execute a snippet of binops until one of them returns true or the commutative pair returns true
+VIREO_FUNCTION_SIGNATURE1(IsLTAccumulator, void)
+{
+    BinaryCompareInstruction* binop = (BinaryCompareInstruction*)_ParamPointer(0);
+    if (DoLTBinayCompareAccumulator(binop)) {
+        return null;
+    }
+    return _this;
+}
+//------------------------------------------------------------
+// Execute a snippet of binops until one of them returns true or the commutative pair returns true
+VIREO_FUNCTION_SIGNATURE1(IsLTSortAccumulator, void)
+{
+    BinaryCompareInstruction* binop = (BinaryCompareInstruction*)_ParamPointer(0);
+    if (DoLTBinayCompareAccumulator(binop)) {
+        return null;
+    }
```

Comment:
I don't think you need to wrap this function.  Instead, revert this and use 
DEFINE_VIREO_FUNCTION_CUSTOM( IsLTSortAccumulator, IsLTAccumulator, "p(i(GenericBinOp))");
below so the LT and LTSort accumulator can be directly shared.